docs/Testing-Simorgh.md

```diff
@@ -0,0 +1,88 @@
+# TESTING FOR ARTICLES
```

Comment:
As this document has a lot of info I think adding a contents list with links to the relevant sections would be helpful 👍